pkg/assetvalidator/validateAssets.go

```diff
@@ -9,22 +9,42 @@ import (
 	"github.com/asecurityteam/nexpose-asset-producer/pkg/domain"
 )
 
+// NexposeScanType is a type alias to signify type of scan according to Nexpose
+type NexposeScanType string
+
+const (
+	automated NexposeScanType = "Automated"
+	manual    NexposeScanType = "Manual"
+	scheduled NexposeScanType = "Scheduled"
+	agent     NexposeScanType = "Agent"
+)
+
+// ScanType is a type alias to signify type generify scan, local or remote
```

Comment:
NIT: What's `generify`?

Reply:
ummm it's just a adverb for `generic`. i dont really like this sentence structure. let me change it.